janis_bioinformatics/tools/dawson/createcallregions/base.py

```diff
@@ -0,0 +1,61 @@
+from janis_core import PythonTool, TOutput, File
+from typing import Dict, Optional, List, Any
+
+from janis_core import Array
+from janis_bioinformatics.data_types import FastaFai
+
+
+class CreateCallRegions(PythonTool):
+    @staticmethod
```

Comment:
Can you add the `friendly_name(self): return "<friendly name>"` method to generate docs correctly.